src/constants.js

```diff
@@ -12,6 +12,7 @@ export const envs = {
 };
 
 // Variables
+export const guildId = '334891772696330241';
```

Comment:
wonder if this should be a database constant.

Reply:
it's not all that sensitive, but maybe yeah

Reply:
or maybe it is sensitive idk

Reply:
either way guess it would be good practice to have it elsewhere, i was lazy